resources/lang/en/events/misc/general.json

```
@@ -1366,17 +1420,21 @@
             "cats_from": ["r_c"],
             "mutual": true,
             "values": ["platonic", "trust", "comfort"],
-            "amount": 10
+            "amount": 15
           }
         ]
       },
     {
-        "event_id": "gen_misc_leafbare_kitkit2",
-        "location": ["any"],
+        "event_id": "multi_misc_leafbare_kitkit2",
+        "location": [
+          "forest",
+          "plains",
+          "mountainous"
+        ],
         "season": ["leaf-bare"],
       "tags": ["clan:warrior"],
         "weight": 20,
-        "event_text": "m_c and r_c flick snow at the older warriors, giggling as they hide behind the nursery.",
+        "event_text": "m_c and r_c flick snow at the adult warriors, giggling as they hide behind the nursery.",
```

tbh I don't think it's necessary to even clarify "adult", I think just saying "warriors" would be fine

---

cool beans, I'll remove it <3
```suggestion
        "event_text": "m_c and r_c flick snow at the warriors, giggling as they hide behind the nursery.",
```